README.md

```diff
@@ -4,17 +4,23 @@
 
 A set of guidelines and best practices for an awesome engineering team.
 Heavily "inspired" by the
-[18-F Development Guide](https://github.com/18f/development-guide).
+[18F Development Guide](https://github.com/18f/development-guide).
 
 * [Open Source Policy](/open-source-policy)
 <!-- * [Development Ethos](/ethos)-->
 * [Project Setup](/project_setup)
-<!-- * [Personal Development Environments](/dev_envs)-->
+* [Development Environments](/dev_envs)
+  * [CISA-provided Mac Setup](/dev_envs/mac-env-setup.md)
 * Language Guides
   * HCL
   * JavaScript
   * [Python](/languages/python)
 
+## Git, GitHub, and You ##
```

Comment:
Do we want to make the capitalization of these headers agree with what we are doing in the other files in this PR?

Reply:
Yup, and I'm fixing the pre-existing headers too.

Reply:
Fixed in a848650149665216a068543dd1b50962d7ec2737